api/migrations/20210812174148-create-guild.js

```diff
@@ -29,6 +30,10 @@ module.exports = {
       moderationServer: {
         type: Sequelize.STRING,
       },
+      modCooldown: {
```

Comment:
What's the unit of this? Is it in seconds, milliseconds or something else?

Reply:
Milliseconds. 
The reason why I have it in milliseconds is because I am going to be comparing unix dates, and that will result in milliseconds.

Reply:
SGTM, can you make it clear that it's milliseconds?